project/Dependencies.scala

```diff
@@ -77,6 +77,7 @@ object Dependencies {
   val scanamo = "com.gu" %% "scanamo" % "0.8.3"
   val enumeratumPlayJson = "com.beachape" %% "enumeratum-play-json" % "1.5.6"
   val commercialShared = "com.gu" %% "commercial-shared" % "6.0.0"
+  val jenkins = "com.gu" %% "jenkins" % "0.1-SNAPSHOT"
```

Comment:
@regiskuckaertz Did you come up with a reason not to integrate this within the CAPI client?